appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/DraggableChildren.kt

```diff
@@ -63,12 +67,13 @@ fun <InteractionTarget : Any, ModelState : Any> DraggableAppyxComponent(
     val coroutineScope = rememberCoroutineScope()
     val gestureExtraTouchAreaPx = with(density) { gestureExtraTouchArea.toPx() }
     var uiContext by remember { mutableStateOf<UiContext?>(null) }
+    var boxScope: BoxScope? = null
 
     LaunchedEffect(uiContext) {
         uiContext?.let { appyxComponent.updateContext(it) }
     }
     Box(
-        modifier = modifier
+        modifier = Modifier
```

Comment:
Input param `modifier` isn't used anymore in this context. Is this intended?

Reply:
Good catch!